kedro/pipeline/node.py

```diff
@@ -550,10 +544,10 @@ def _validate_inputs(self, func, inputs):
                 func_args = inspect.signature(
                     func, follow_wrapped=False
                 ).parameters.keys()
+                func_name = _get_readable_func_name(func)
+
                 raise TypeError(
-                    "Inputs of function expected {}, but got {}".format(
-                        str(list(func_args)), str(inputs)
-                    )
+                    f"Inputs of '{func_name}' function expected {list(func_args)}, but got {inputs}"
```

Comment:
This line is probably longer than 88 characters, so needs to be broken down in 2

Reply:
good catch